app/models/package_manager/npm.rb

```diff
@@ -40,11 +40,11 @@ def self.project(name)
     end
 
     def self.deprecation_info(name)
-      versions = project(name)["versions"].values
+      last_version = project(name)["versions"].values.last
```

Comment:
We sure these are in timeorder? Could max_by just to be sure.

Reply:
We were using timeorder before to get the deprecation message, and in my experience they've been ordered. But I'll take a quick look to see if we can sort by semver to be sure.

Reply:
If they generally seem like they're in the right order coming in, I'd trust that over an attempted semver sort 🤣